File: largest number.cpp
#include <iostream>
using namespace std;
int main() {
    int num1, num2, num3;

    cout << "Enter three numbers: ";
    cin >> num1 >> num2 >> num3;

    int largest; 

    if (num1 >= num2 && num1 >= num3) {
        largest = num1;
    } else if (num2 >= num1 && num2 >= num3) {
        largest = num2;
    } else {
        largest = num3;
    }

    cout << "The largest of the three is: " << largest << endl;

    return 0;
}

File: largest number.py
a=int(input("Enter first number:"))
b=int(input("Enter first number:"))
c=int(input("Enter first number:"))

if(a>=b)and(a>=c):
    largest=a
elif(b>=a)and(b>=c):
    largest=b
else:
    largest=c

print("The largest number is",largest)